icons/wallet.svg

```diff
@@ -0,0 +1,13 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M3 6V18C3 19.1 3.9 20 5 20H19V16M3 6C3 7.1 3.9 8 5 8H17M3 6C3 4.9 3.9 4 5 4H17V8M17 8H19V12M15 14C15 12.9 15.9 12 17 12H21V16H17C16.4696 16 15.9609 15.7893 15.5858 15.4142C15.2107 15.0391 15 14.5304 15 14Z" />
```

Comment:
Split single path in pieces
```suggestion
  <svg
  xmlns="http://www.w3.org/2000/svg"
  width="24"
  height="24"
  viewBox="0 0 24 24"
  fill="none"
  stroke="currentColor"
  stroke-width="2"
  stroke-linecap="round"
  stroke-linejoin="round"
>
  <path d="M20 12V8H6a2 2 0 01-2-2c0-1.1.9-2 2-2h12v4" />
  <path d="M4 6v12c0 1.1.9 2 2 2h14v-4" />
  <path d="M18 12a2 2 0 00-2 2c0 1.1.9 2 2 2h4v-4h-4z" />
```

Reply:
> Split single path in pieces

Should we add this to Lucide icon guidelines?

@ericfennis @locness3 @johnletey